### PR TITLE
Remove Projects

### PR DESCRIPTION
**Pull request by movchan74**
**Summary**:

This update simplifies the SDK by removing projects, relocating specific applications to separate repositories, and refining the testing approach. It aims to streamline the codebase and improve maintainability.

**Key Changes:**
1. **Project Removal**: Removed projects from the SDK. "Summarize Transcript" app is now at https://github.com/mobiusml/aana_summarize_video/ and "Chat with Video" app is at https://github.com/mobiusml/aana_chat_with_video/. The 'lowercase' app is now used in tests/projects for integration testing.

2. **Test Modifications**: Removed integration tests for apps and their expected endpoint results.

3. **Testing Refinement**: Simplified integration tests to validate output schema instead of exact matches. 

4. **Entity Relocation**: Moved ExtendedVideoEntity, ExtendedVideoTranscriptEntity, ExtendedVideoCaptionEntity, and their repositories to the "Chat with Video" project.

5. **Documentation Update**: Updated documentation to reflect all changes made in this update.

**Related PRs**: https://github.com/mobiusml/aana_summarize_video/pull/1, https://github.com/mobiusml/aana_chat_with_video/pull/1

**Related issues**: https://github.com/mobiusml/aana_sdk/issues/129

### CODE REVIEW

**Comment by HRashidi** [comment]
We may need to remove the cache files too. At least most of them.

**Comment by movchan74** [comment]
> We may need to remove the cache files too. At least most of them.

@HRashidi I plan to remove the test cache later as a separate PR. We don't need it anymore.

File: aana/tests/units/test_app.py [hunk 1]
```diff
@@ -76,9 +76,9 @@ async def run(self, text: TextList) -> LowercaseEndpointOutput:
 ]
 
 
-def test_app(app_setup):
+def test_app(app_create):
     """Test the Ray Serve app."""
-    aana_app = app_setup(deployments, endpoints)
+    aana_app = app_create(deployments, endpoints)
```
Comment (HRashidi) [inline aana/tests/units/test_app.py:81]: create_app is a better name!

File: aana/utils/core.py [hunk 2]
```diff
@@ -62,3 +62,19 @@ def import_from_path(path, reload=False):
...
+
+
+def get_module_dir(module_name: str) -> Path:
+    """Get the directory of a module.
+
+    Args:
+        module_name (str): Name of the module (e.g. 'aana')
+
+    Returns:
+        Path: Root directory of the module.
```
Comment (HRashidi) [inline aana/utils/core.py:74]: Parent directory of a module?

File: docs/pages/tutorial.md [hunk 3]
```diff
@@ -594,8 +594,58 @@ import requests, json;
...
+CUDA_VISIBLE_DEVICES=0 poetry run aana deploy aana_summarize_video.app:aana_app
+```
+
+Once the application is running, you can send a request to transcribe and summarize a video as described in the previous sections.
+
+
 ## Conclusion
 
-In this tutorial, we have walked you through the process of creating a new project with Aana SDK. We have reviewed the video transcription application and extended it to include the LLM model for summarization and a new endpoint. We have also demonstrated how to stream the output from the LLM model. You can use this tutorial as a reference to build your own applications with Aana SDK. 
+In this tutorial, we have walked you through the process of creating a new project with Aana SDK. We have reviewed the video transcription application and extended it to includ e the LLM model for summarization and a new endpoint. We have also demonstrated how to stream the output from the LLM model. You can use this tutorial as a reference to build your own applications with Aana SDK.
```
Comment (HRashidi) [inline docs/pages/tutorial.md:649]: typo

File: aana/sdk.py [hunk 4]
```diff
@@ -86,7 +90,17 @@ def connect(
 
     def migrate(self):
         """Run Alembic migrations."""
-        run_alembic_migrations(aana_settings)
+        print(aana_settings.db_config)
```
Comment (HRashidi) [inline aana/sdk.py:93]: wanna keep the print?

File: docs/pages/tutorial.md [hunk 5]
```diff
@@ -594,8 +594,58 @@ import requests, json;
...
+cd <repository-name>
+```
+
+### Create the Application
+
+For our example project we need to adjust a few things.
+
+#### Deployments
+
+Define the deployments in the [`aana_summarize_video/configs/deployments.py`](https://github.com/mobiusml/aana_summarize_video/blob/main/aana_summarize_video/configs/deployments.py). Just like in the example above.
```
Comment (HRashidi) [inline docs/pages/tutorial.md:622]: It's better to mention the file configs/deployments.py then pointed to the example

File: aana/utils/core.py (hunk 2 above)
Comment (movchan74) [reply aana/utils/core.py:74]: No, root.

File: README.md [hunk 6]
```diff
@@ -196,39 +196,13 @@ This will return the full transcription of the video, transcription for each seg
...
-- `chat_with_video`: A multimodal chat application that allows users to upload a video and ask questions about the video content based on the visual and audio information. See [Chat with Video Demo notebook](/notebooks/chat_with_video_demo.ipynb) for more information.
-- `whisper`: An application that demonstrates the Whisper model for automatic speech recognition (ASR).
-- `llama2`: An application that deploys LLaMa2 7B Chat model.
+- [Chat with Video](https://github.com/mobiusml/aana_chat_with_video): A multimodal chat application that allows users to upload a video and ask questions about the video content based on the visual and audio information. See [Chat with Video Demo notebook](https://github.com/mobiusml/aana_chat_with_video/blob/main/notebooks/chat_with_video_demo.ipynb) to see how to use the application.
+- [Summarize Video](https://github.com/mobiusml/aana_summarize_video): An Aana application that summarizes a video by extracting transcription from the audio and generating a summary using a Language Model (LLM). This application is a part of the [tutorial](https://mobiusml.github.io/aana_sdk/pages/tutorial/) on how to build multimodal applications with Aana SDK.
```
Comment (HRashidi) [inline README.md:202]: reminder: The projects are still private.

**Review by HRashidi — COMMENTED** [review]
Some comments

File: README.md (hunk 6 above)
Comment (movchan74) [reply README.md:202]: I will make them public once we merge the PRs.

File: docs/pages/tutorial.md (hunk 5 above)
Comment (movchan74) [reply docs/pages/tutorial.md:622]: I'm not sure I understand. Can you clarify what you mean? The example is made for this tutorial, it's just the code from the tutorial as an Aana App project. So pointing to the deployments.py I want to show what that transition (code from the tutorial -> app project) looks like. Keep in mind that `configs/deployments.py` here is not the same as `configs/deployments.py` in the Aana SDK. `configs/deployments.py` in the Aana SDK is about to be deleted.

File: aana/tests/units/test_app.py (hunk 1 above)
Comment (movchan74) [reply aana/tests/units/test_app.py]: Done

File: aana/sdk.py (hunk 4 above)
Comment (movchan74) [reply aana/sdk.py]: Removed

File: docs/pages/tutorial.md (hunk 3 above)
Comment (movchan74) [reply docs/pages/tutorial.md]: Fixed

File: docs/pages/tutorial.md (hunk 5 above)
Comment (HRashidi) [reply docs/pages/tutorial.md:622]: something like this:
`define your deployments in [configs/deployments.py](https://github.com/mobiusml/aana_app_template/blob/main/aana_app_project/configs/deployments.py) file like this example [`aana_summarize_video/configs/deployments.py`](https://github.com/mobiusml/aana_summarize_video/blob/main/aana_summarize_video/configs/deployments.py)

**Review by evanderiel — APPROVED** [review]

File: docs/pages/tutorial.md (hunk 5 above)
Comment (movchan74) [reply docs/pages/tutorial.md]: Done. Can you take a look now?

**Review by HRashidi — APPROVED** [review]

**Pull request merged**